src/plots/geo/layout_defaults.js

```diff
@@ -46,6 +46,29 @@ function handleGeoDefaults(geoLayoutIn, geoLayoutOut, coerce, opts) {
     var isClipped = geoLayoutOut._isClipped = !!constants.lonaxisSpan[projType];
 
     var visible = coerce('visible');
+    if(geoLayoutIn.visible === false) {
+        visible = geoLayoutOut.visible = true;
```

Comment:
Why are we setting `geoLayoutOut.visible = true` here? Looks to me as though it doesn't matter for the end result, is that right? RCE would want the `false` to remain intact.

Reply:
My bad.